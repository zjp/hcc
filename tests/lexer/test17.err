FATAL [1,1]: Empty escape sequence in character literal
