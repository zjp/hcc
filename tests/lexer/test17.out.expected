EOF [2,1]
